util/LOGGING.md

```diff
@@ -78,4 +88,6 @@ Variable                    | Corresponding environment var
 `LogOnScreen`               | `GOVCD_LOG_ON_SCREEN`
 `LogHttpRequest`            | `GOVCD_LOG_SKIP_HTTP_REQ`
 `LogHttpResponse`           | `GOVCD_LOG_SKIP_HTTP_RESP`
+`SkipTagList`               | `GOVCD_LOG_SKIP_TAG_LIST`
+`IncludeFunctionList`       | `GOVCD_LOG_INCLUDE_FUNCTION_LIST`
```

Comment:
I am not so sure about IncludeFunctionList - at least for me include means add aditionally what it is happening.
I would say better naming would be "OnlyFucntionList"

Reply:
I use `include` as opposed to `skip`. I don't like using `only` in this context because it is not a verb.
However, if there is consensus, I will change the name to `OnlyFunctionList`.

Reply:
'ShowOnlyFunctionList' then maybe. I am open for better naming.

Reply:
fixed using Linas's suggestion